assets/misc/ebnf-filtering.txt

```diff
@@ -0,0 +1,184 @@
+# Filter, possibly empty
```

Comment:
This is directly lifted from @TristonianJones. Note that we will be publishing it publicly if we accept this (which I think is fine).

Reply:
Did you want it as Markdown or just plain text?

Reply:
Inclined to say plain text, but I can be talked into something else.